extensions/cornerstone/src/initMeasurementService.ts

```diff
@@ -461,6 +461,7 @@ const connectMeasurementServiceToTools = ({
       const annotationManager = annotation.state.getAnnotationManager();
       const newAnnotation = {
         annotationUID: measurement.uid,
+        predecessorImageId: measurement?.predecessorImageId,
```

Comment:
We don't have this in Cornerstone, and you're adding it as if it exists. It might be just for handling state, but please add a comment above it.